source/Sandbox.JobServer/Sandbox.JobServer.csproj

```diff
@@ -1,107 +1,41 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{28B3BFFA-26E2-4A0E-A329-B6FD729FB981}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <RootNamespace>Sandbox.JobServer</RootNamespace>
-    <AssemblyName>Sandbox.JobServer</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFramework>net472</TargetFramework>

```

Comment:
What about this? 🤔

Reply:
These are just "executable" samples which don't necessary have to be upgrade.
But I can upgrade them to netcoreapp2.0